_data/sidebars/zh_lb2_sidebar.yml

```diff
@@ -1,5 +1,899 @@
 entries:
-- title: sidebar
-  product: LoopBack - 中文
-  home_url: /doc/zh/lb2/index.html
-  folders:
+
```

Comment:
I'll review and merge your PR asap.  Then, we'll have to rework these sidebars before merging this new Chinese content.